odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -3172,6 +3172,21 @@ def test_01_sanitize_groups(self):
         self.assertEqual(record.comment5, '',
                          "should be sanitized (not in groups)")
 
+    def test_read_sanitize_overridable(self):
+        self.assertTrue(self.registry['test_new_api.mixed'].comment5.sanitize_overridable)
+
+        internal_user = self.env['res.users'].create({
+            'name': 'test internal user',
+            'login': 'test_sanitize',
+            'groups_id': [(6, 0, [self.ref('base.group_user')])],
+        })
+        record = self.env['test_new_api.mixed'].with_user(internal_user).create({
+            'comment5': '<p>comment</p>',
+        })
+        self.assertEqual(record.comment5, '<p>comment</p>')
+        new_record = record.new(origin=record)
+        self.assertEqual(new_record.comment5, '<p>comment</p>')
```

Comment:
Please add a comment before that line that explains the purpose of the test.  Something like
```py
# this field access was causing an infinite loop in HTML sanitization
```

Reply:
Could you move both test methods next to each other?
You can move both of them in `test_new_fields.py` (the tests are not strictly about `onchange`).
This will improve readability.